src/app/graphql/queries/user-info.gql

```diff
@@ -0,0 +1,28 @@
+query (
+  $searchName: String!
```

Comment:
@bhoomika08 use case of !?

Reply:
@gurpreet013 ! used when the value is non-nullable and it is required. 
We have to give the value of argument $searchName in this case.